dotnet/src/webdriver/JavaScriptEngine.cs

```diff
@@ -219,14 +219,25 @@ public async Task ClearInitializationScripts()
         /// </summary>
         /// <param name="script">The JavaScript to pin</param>
         /// <returns>A task containing a <see cref="PinnedScript"/> object to use to execute the script.</returns>
+        /// <exception cref="ArgumentNullException">If <paramref name="script"/> is <see langword="null"/>.</exception>
         public async Task<PinnedScript> PinScript(string script)
         {
+            if (script == null)
+            {
+                throw new ArgumentNullException(nameof(script));
+            }
+
+            string newScriptHandle = Guid.NewGuid().ToString("N");
```

Comment:
Seems it is new logic. What is a reference we can point to consider it as not a breaking change?

Reply:
It is populated now at layer upper.